Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -253,10 +253,11 @@ def genGridOnPoints(data1, gm, deep=True, grid=None, geo=None,
     if geo is None:
         bounds = pts.GetBounds()
         xm, xM, ym, yM = [bounds[0], bounds[1], bounds[2], bounds[3]]
-        # we use plotting coordinates for doing the projection
+        # We don't use the zooming feature (gm.datawc) for geographic
+        # projections. We use wrapped coordinates for doing the projection
         # such that parameters such that central meridian are set correctly
-        geo, geopts = project(pts, projection, getBoundsForPlotting(
-            [gm.datawc_x1, gm.datawc_x2, gm.datawc_y1, gm.datawc_y2], [xm, xM, ym, yM], wrap))
+        geo, geopts = project(pts, projection, getWrappedBounds(
+            [gm.datawc_x1, gm.datawc_x2, gm.datawc_y1, gm.datawc_y2], [xm, xM, ym, yM], None))
```

Comment:
why wrap None here? Does geo = None means plain array or does it means no projection? I can't remember for sure.